server/services/FiltersService/index.js

```diff
@@ -66,149 +66,40 @@ class FiltersService {
-                    included_excluded: { $in: includedExcludedFilters },
-                  },
-                },
-              },
-            ],
-            study: { $in: this.filters.sites },
-          },
-        },
-      ]
-    }
-    if (isIncOrExcOnly || isCritOnly) {
-      const variable = chrCritFilters.length
-        ? 'chrcrit_part'
-        : 'included_excluded'
-      const values = chrCritFilters.length
-        ? chrCritFilters
-        : includedExcludedFilters
-
-      $facet.formInclusion = [
-        {
-          $match: {
-            assessment: INCLUSION_EXCLUSION_CRITERIA_FORM,
-            study: {
-              $in: this.filters.sites,
-            },
-            dayData: {
-              $elemMatch: { [variable]: { $in: values } },
-            },
-          },
-        },
-      ]
-    }
-    if (sexAtBirthFilters.length) {
-      $facet.sexAtBirth = [
-        {
-          $match: {
-            assessment: SOCIODEMOGRAPHICS_FORM,
-            dayData: {
-              $elemMatch: {
-                chrdemo_sexassigned: { $in: sexAtBirthFilters },
-              },
-            },
-          },
-        },
-      ]
-    }
-    return Object.keys($facet).length > 1
-      ? [
-          { $facet },
-          {
-            $project: {
-              participants: {
-                $setIntersection: [
-                  '$formInclusion.participant',
-                  '$sexAtBirth.participant',
-                ],
-              },
-            },
-          },
-        ]
-      : $facet
-  }
-
-  _buildFacetForFilters = ({
-    isSexAtBirthFilterActive,
-    isInclusionCriteriaFilterActive,
-    sites,
-  }) => {
-    const facetForFilters = {}
-    if (isSexAtBirthFilterActive) {
-      facetForFilters.socioDemographics = [
-        {
-          $match: {
-            assessment: SOCIODEMOGRAPHICS_FORM,
-            study: { $in: sites },
-          },
-        },
-        {
-          $project: {
-            ...INDIVIDUAL_FILTERS_MONGO_PROJECTION,
-          },
-        },
-        {
-          $addFields: {
-            filter: SOCIODEMOGRAPHICS_FORM,
-          },
-        },
-      ]
-    }
-    if (isInclusionCriteriaFilterActive) {
-      facetForFilters.inclusionCriteria = [
-        {
-          $match: {
-            assessment: INCLUSION_EXCLUSION_CRITERIA_FORM,
-            study: { $in: sites },
-          },
-        },
-        {
-          $project: {
-            ...INDIVIDUAL_FILTERS_MONGO_PROJECTION,
-          },
-        },
-        {
-          $addFields: {
-            filter: INCLUSION_EXCLUSION_CRITERIA_FORM,
-          },
-        },
-      ]
+              {assessment: INCLUSION_EXCLUSION_CRITERIA_FORM},
+              {'dayData.included_excluded': {$in: includedValues } }
+            ]
+          })
+          break
+        case 'chrcrit_part':
+          filterQueries.push({
+            $and: [
+              {assessment: INCLUSION_EXCLUSION_CRITERIA_FORM},
```

Comment:
another nit on formatting. Can you run prettier?

Reply:
We don't have prettier installed or configured for this app.

Reply:
[formatting fixes](https://github.com/AMP-SCZ/dpdash/pull/659/commits/ad8bf5c42ecf247244eda6a06081123d225c4863)